test/e2e/.babelrc

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
Splitting configs to multiple files is always unfortunate instead of having one file with visibility to all configs at once as we have with our eslint file.

via https://babeljs.io/docs/en/config-files#monorepos

>Separately, it is also important to decide if you want to use .babelrc files or just a central babel.config.js. .babelrc files are not required for subfolder-specific configuration like they were in Babel 6, so often they are not needed in Babel 7, in favor of a babel.config.js.

Could this be moved to root config under "overrides"?

Reply:
I tried this and wasn't able to get it to work. I think it is because the tests are still being run from the new subdirectory rather than the root, so the babel.config.js isn't checked. I can look in to this further for a future iteration

Reply:
Babel looks for the `babel.config.js` config file only in the current working directory, so the file in `..` is not used. There's a `rootMode: 'upward'` option [to override that](https://babeljs.io/docs/en/config-files#root-babelconfigjs-files) in monorepos. Would it be useful for us?

Why do we use Babel at all? Is there any JSX or other non-standard syntax to transpile? Running on recent 10.x Node has the benefit of full support for latest standard JS without transpilation.